github-management/kubernetes-repositories.md

```diff
@@ -98,6 +98,13 @@ the developers who could not be reached
      [@caniszczyk](https://github.com/caniszczyk) for review of third party deps
    * Boilerplate text across all files should attribute copyright as follows:
      `"Copyright <Project Authors>"` if no CLA was in place prior to donation
+   * Additions of [the standard Kubernetes header](https://github.com/kubernetes/kubernetes/blob/master/hack/boilerplate/boilerplate.go.txt)
```

Comment:
The line above says `Copyright <Project Authors>`, can we modify that to point to the standard Kubernetes header as well?